### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by dominikkiese**

### CODE REVIEW

File: src/Action/action_lib/action_u1_dm.jl [hunk 1]
```diff
@@ -210,6 +210,84 @@ function symmetrize_add_to!(
...
+                a.Γ[3].ch_u.q2_1[i, w, v] += a_l.Γ[3].ch_u.q2_1[i, w, v] + a_l.Γ[3].ch_u.q2_2[i, w, v]
+                a.Γ[4].ch_u.q2_1[i, w, v] += a_l.Γ[4].ch_u.q2_1[i, w, v] + a_l.Γ[4].ch_u.q2_2[i, w, v]
+                a.Γ[5].ch_u.q2_1[i, w, v] += a_l.Γ[5].ch_u.q2_1[i, w, v] - a_l.Γ[5].ch_u.q2_2[i, w, v]
+                a.Γ[6].ch_u.q2_1[i, w, v] += a_l.Γ[6].ch_u.q2_1[i, w, v] - a_l.Γ[6].ch_u.q2_2[i, w, v]
+                a.Γ[1].ch_u.q2_2[i, w, v] += a_l.Γ[1].ch_u.q2_2[i, w, v] + a_l.Γ[1].ch_u.q2_1[i, w, v]
+                a.Γ[2].ch_u.q2_2[i, w, v] += a_l.Γ[2].ch_u.q2_2[i, w, v] + a_l.Γ[2].ch_u.q2_1[i, w, v]
+                a.Γ[3].ch_u.q2_2[i, w, v] += a_l.Γ[3].ch_u.q2_2[i, w, v] + a_l.Γ[3].ch_u.q2_1[i, w, v]
+                a.Γ[4].ch_u.q2_2[i, w, v] += a_l.Γ[4].ch_u.q2_2[i, w, v] + a_l.Γ[4].ch_u.q2_1[i, w, v]
+                a.Γ[5].ch_u.q2_2[i, w, v] += a_l.Γ[5].ch_u.q2_2[i, w, v] - a_l.Γ[5].ch_u.q2_1[i, w, v]
+                a.Γ[6].ch_u.q2_2[i, w, v] += a_l.Γ[6].ch_u.q2_2[i, w, v] - a_l.Γ[6].ch_u.q2_1[i, w, v]
```
Comment (tobiaslcmueller) [inline src/Action/action_lib/action_u1_dm.jl:286]: Just for me: is there an easy way to see, that zd does not become dz when exchanging v and v'?

**Review by tobiaslcmueller — APPROVED** [review]
Symmetrize_add_to now seems to work, results look reasonable.
Good to go from my side

File: src/Action/action_lib/action_u1_dm.jl (hunk 1 above)
Comment (lgresista) [reply src/Action/action_lib/action_u1_dm.jl:286]: I don't think there is an easy way to see this. A short explanation would be:
The v <-> v' symmetry originates from either the s -> -s, t -> -t, or u -> -u symmetry, depending on the channel. In the u-channel it originates from the t -> -t symmetry, which unlike the s -> -s and the u -> -u symmetry, does not come with a site and spin exchange (see Finn's paper). Therefore there is also no mapping of dz to zd and xy -> yx.

**Review by lgresista — APPROVED** [review]
I think everything is correct. Ready to merge.

**Pull request merged**